src/components/Social.js

```diff
@@ -1,35 +1,35 @@
 import React from "react"
 
-import ytLogo from "../images/youtube_social_icon_red.png"
-import twLogo from "../images/Twitter_Social_Icon_Rounded_Square_Color.png"
+import YoutubeIcon from "../images/youtube_social_icon_red.png"
+import TwitterIcon from "../images/Twitter_Social_Icon_Rounded_Square_Color.png"
 import { TiledLink, TiledLinkContainer, TiledLinkIcon, SectionTitle } from '../theme'
 
 const Social = () => (
   <div className="social">
     <SectionTitle>Socials</SectionTitle>
     <TiledLinkContainer>
       <TiledLink href="https://www.youtube.com/channel/UCuXhrIE_d-Soapk2WEuGXAg">
-        <TiledLinkIcon src={ytLogo} alt="Unhandled Exception Club Youtube" />
+        <TiledLinkIcon src={YoutubeIcon} alt="Unhandled Exception Club Youtube" />
         Follow Unhandled Exception on Youtube
       </TiledLink>
 
       <TiledLink href="https://twitter.com/eralpkaraduman">
-        <TiledLinkIcon src={twLogo} alt="Eralp Karaduman Twitter" />
+        <TiledLinkIcon src={TwitterIcon} alt="Eralp Karaduman Twitter" />
         @eralpkaraduman
```

Comment:
By convention `@` signs should be `&#64;` in html, here it works fine because jsx is preprocessed. However would be nice to follow the convention. To avoid using html entities even better solution could be replacing `&#64;eralpkaraduman` with `{'@eralpkaraduman'}`